build.py

```diff
@@ -410,19 +410,12 @@ def tensorflow_cmake_args(ver, images, library_paths):
                     library_paths[backend_name])
             ]
     else:
-        # If a specific TF image is specified use it, otherwise pull from NGC.
-        backend_name = "tensorflow{}".format(ver)
-        if backend_name in images:
-            image = images[backend_name]
-        else:
-            if ver == 2:
-                image = 'nvcr.io/nvidia/tensorflow:{}-py3'.format(
-                    FLAGS.upstream_container_version)
-            else:
-                image = 'nvcr.io/nvidia/tensorflow:{}-tf{}-py3'.format(
-                    FLAGS.upstream_container_version, ver)
-        extra_args = ['-DTRITON_TENSORFLOW_DOCKER_IMAGE={}'.format(image)]
-    return ['-DTRITON_TENSORFLOW_VERSION={}'.format(ver)] + extra_args
+        image = 'nvcr.io/nvidia/tensorflow:{}-tf{}-py3'.format(
```

Comment:
I think this change removes support for non NGC TF docker images. Is this intended?

Reply:
Yes, there is some merge error here. The following is the correct function. @CoderHam please verify.

def tensorflow_cmake_args(ver, images, library_paths):
    backend_name = "tensorflow{}".format(ver)

    # If platform is jetpack do not use docker images
    extra_args = []
    if target_platform() == 'jetpack':
        if backend_name in library_paths:
            extra_args = [
                '-DTRITON_TENSORFLOW_LIB_PATHS={}'.format(
                    library_paths[backend_name])
            ]
    else:
        # If a specific TF image is specified use it, otherwise pull from NGC.
        if backend_name in images:
            image = images[backend_name]
        else:
            if ver == 2:
                image = 'nvcr.io/nvidia/tensorflow:{}-py3'.format(
                    FLAGS.upstream_container_version)
            else:
                image = 'nvcr.io/nvidia/tensorflow:{}-tf{}-py3'.format(
                    FLAGS.upstream_container_version, ver)
        extra_args = ['-DTRITON_TENSORFLOW_DOCKER_IMAGE={}'.format(image)]
    return ['-DTRITON_TENSORFLOW_VERSION={}'.format(ver)] + extra_args

Reply:
Yes. This is the correct. @dzier could you fix the function to be the same as above?